tm-tool/textmapper/resources/org/textmapper/tool/templates/go_parser.ltp

```diff
@@ -294,11 +286,11 @@ ${end-}
 				}
 			}
 ${if self->ignoredReportTokens()-}
```

Comment:
I think flushing the pending tokens here is a bug. If the following recoverFromError recovers without shifting any tokens, and then we reduce `SyntaxProblem: error -> SyntaxProblem;`, this SyntaxProblem should be reported before any subsequent comments.

We should flush p.pending in front of every fetchNext and when we put noToken into p.next.symbol.

Reply:
Done.